CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# How to Contribute to HugeGraph
+
+Thanks for taking the time to contribute! As an open source project, HugeGraph is looking forward to be contributed from everyone, and we are also grateful to all of the contributors.
+
+The following is a guide for the contribution process of HugeGraph:
+
+## 1. Preparation
+
+We can contribute by reporting issue or submitting code patch or any other feedback.
```

Comment:
We can contribute by reporting issues, submitting code patches or any other feedback.